admin-ui/plugins/fido/types/types.ts

```diff
@@ -1,6 +1,161 @@
+  hints?: string[]
+  enterpriseAttestation?: boolean
+  attestationMode?: string
+  rp?: RequestedParty[]
+}
+
+export interface AppConfiguration {
+  issuer?: string
+  baseEndpoint?: string
+  cleanServiceInterval?: number
+  cleanServiceBatchChunkSize?: number
+  useLocalCache?: boolean
+  disableJdkLogger?: boolean
+  loggingLevel?: string
+  loggingLayout?: string
+  externalLoggerConfiguration?: string
+  metricReporterInterval?: number
+  metricReporterKeepDataDays?: number
+  metricReporterEnabled?: boolean
+  personCustomObjectClassList?: string[]
+  sessionIdPersistInCache?: boolean
+  errorReasonEnabled?: boolean
+  fido2Configuration?: Fido2Configuration
+}
+
+export interface PutPropertiesFido2Options {
+  appConfiguration?: AppConfiguration
+}
+
+export interface SmtpTest {
+  sign?: boolean
+  subject?: string
+  message?: string
+}
+
+export interface SearchFido2RegistrationOptions {
+  limit?: number
+  pattern?: string
+  startIndex?: number
+  sortBy?: string
+  sortOrder?: string
+  fieldValuePair?: string
+}
+
+export interface DeleteFido2DeviceInput {
+  jansId: string
+}
+
+export interface TestSmtpConfigInput {
+  smtpTest: SmtpTest
+}
+
+export interface IFido2ConfigurationApi {
+  getPropertiesFido2(callback: ApiCallback<AppConfiguration>): void
+  putPropertiesFido2(
+    opts: { appConfiguration?: AppConfiguration },
+    callback: ApiCallback<AppConfiguration>,
+  ): void
+}
+
+export interface IFido2RegistrationApi {
+  deleteFido2Data(jansId: string, callback: ApiCallback<void>): void
+}
+
+export interface IFido2Api {
+  getPropertiesFido2(): ApiPromise<AppConfiguration>
+  putPropertiesFido2(input: PutPropertiesFido2Options): ApiPromise<AppConfiguration>
+  deleteFido2DeviceData(input: DeleteFido2DeviceInput): ApiPromise<void>
+}
+
+export type ApiCallback<T> = (error: Error | null, data?: T, response?: unknown) => void
+export type ApiPromise<T> = Promise<T>
+
+// Define local RootState interface for this fido saga
+export interface RootState {
+  authReducer: {
+    token: {
+      access_token: string
+    }
+    issuer: string
+    userinfo_jwt: string | null
+  }
+}
+export interface UpdateFidoAction {
+  type: string
+  payload: AppConfiguration
+}
+
+export interface DeleteFido2DeviceAction {
+  type: string
+  payload: DeleteFido2DeviceInput
+}
+
+export interface ApiError extends Error {
+  response?: {
+    data?: {
+      description?: string
+      message?: string
+    }
+  }
+}
+
+export interface ErrorToastAction {
+  error: ApiError
+}
+
+export interface FidoState {
+  fido: AppConfiguration
+  loading: boolean
+}
+
+export interface FidoRootState {
+  fidoReducer: FidoState
+}
+
+export interface TabName {
+  name: string
+  path: string
+}
+
+export interface FidoConfigurationProps {
+  fidoConfiguration: FidoState
+  handleSubmit: (data: FormData) => void
+}
+
+export interface FormData {
+  [key: string]: string | number | boolean | string[] | KeyValuePair[]
+}
```

Comment:
Rename local FormData to FidoFormData across FIDO plugin

- In admin-ui/plugins/fido/types/types.ts, rename the `interface FormData` to `FidoFormData` and update `FidoConfigurationProps.handleSubmit` to accept `FidoFormData`.  
- In admin-ui/plugins/fido/types/index.ts, replace the exported/imported `FormData` with `FidoFormData`.  
- In admin-ui/plugins/fido/components/types/StaticConfiguration.ts and DynamicConfiguration.ts, update imports and `handleSubmit(values: …)` signatures to use `FidoFormData`.  
- In admin-ui/plugins/fido/components/Fido.tsx, import `FidoFormData` instead of `FormData`.